tools/federation/src/test/java/demos/GettingStartedDemoMinimal.java

```diff
@@ -0,0 +1,42 @@
+/*******************************************************************************
+ * Copyright (c) 2019 Eclipse RDF4J contributors.
```

Comment:
No need to fix in this PR, but you may want to look at your template and update the year.

Reply:
Done, I adjusted my template. Thanks for paying attention